less/sortableTable.less

```diff
@@ -15,11 +15,6 @@ table.sort {
 }
 
 table.sortableTable {
-  width: 100%;
```

Comment:
This is replaced with https://github.com/brave/browser-laptop/pull/10265/files#diff-159ff36704e059402c3f17fb8a35a6fcR489. The width is no longer specified by default (since it had to be overwritten with `!important`, which is redundant).